### DISCUSSION TITLE
实现股票图

### DISCUSSION THREAD

**Discussion by usekevinwu** [antvis/G2Plot#3668] [Ideas]
可以实现股票图，选择项，1日，5日，20日，30日，60日。股票图有成交量、5日均线，10日均线，20日均线，30日均线，60日均线。
有点像echarts的
![企业微信截图_16950186091335](https://github.com/antvis/G2Plot/assets/37398179/cef1388c-3d25-4343-af58-d47a37104e8d)

**Comment by hustcc** [comment]
可以基于 [G2](https://g2.antv.antgroup.com/) 在官网帮忙做一个 demo。
